docker-compose.yml

```diff
@@ -38,18 +38,61 @@ services:
       - qdrant
       - ollama
 
-  ragapp:
+  ragapp-cs50:
     build: .
     image: ragapp/ragapp:latest
-    ports:
-      - "8000:8000"
+    entrypoint: ["/app/entrypoint.sh"]
+    command: ["fastapi", "run", "main.py"]
+    volumes:
+      - ./config:/app/config
+      - ./data:/app/data 
+    labels:
+      - ragapp.app_name=cs50
+      - traefik.enable=true
+      - traefik.http.services.ragapp-cs50.loadbalancer.server.port=8000
+      - traefik.http.routers.ragapp-cs50.rule=PathPrefix(`/a/cs50`)
+      - traefik.http.middlewares.ragapp-cs50-strip-path.stripprefix.prefixes=/a/cs50
+      - traefik.http.routers.ragapp-cs50.middlewares=ragapp-cs50-strip-path
```

Comment:
I'm having issue (499 error) with authentication for a prefix routers,example rule=PathRegex(/a/.*/admin) for all app admin pages. So we must specify the rule for each application for now